### PR TITLE
Portfolio: Mobile Version

### PR DESCRIPTION
**Pull request by kelomo2502**
**In this milestone, I created a mobile porfolio website with lists of my projects by following the requirements.**

- Linters are correctly set-up
- Add a descriptive README.md
- HTML and CSS best practices are adhered to

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
+        <div class="posts-container">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="works-btn">
+                <button type="button">css</button>
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+            </div>
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

File: index.html [hunk 2]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
+        <br>
+        <br>
+          <div class="works-cards">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="works-card-btn">
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+          </div>
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

File: index.html [hunk 3]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
+    
+        <div class="works-cards">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+                standard</p>
+            <div class="works-card-btn">
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+            </div>
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

File: index.html [hunk 4]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
+        </div>
+        <div class="works-cards">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+                standard</p>
+            <div class="works-card-btn">
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+            </div>
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

File: index.html [hunk 5]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

File: index.html [hunk 6]
```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
...
```
Comment (Lucas-Masaba) [inline index.html]: - You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @kelomo2502  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!


almost-there

## Highlights 🍾 
- The UI follows that of the Figma 😎 
- Linter checks are passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- I encourage you to write more commit messages for each new functionality you add. Currently you have one commit message for all the features you added as seen here 
image

**Review by codepantha — APPROVED** [review]
Hi @kelomo2502 👋,

## STATUS: APPROVED ✔️  

Weldone implementing the required changes 👏 👏 
Your project is now complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**